src/Symfony/Component/Yaml/Inline.php

```diff
@@ -161,8 +161,8 @@ public static function dump($value, int $flags = 0): string
                 return 'true';
             case false === $value:
                 return 'false';
-            case ctype_digit($value):
-                return \is_string($value) ? "'$value'" : (int) $value;
```

Comment:
We still need to handle the case of `is_string($value) && `ctype_digit($value)`, to dump the value quoted.

Reply:
this case is handled L180, isn't it?

Reply:
indeed, this case should be covered by `is_numeric($value)`.

Btw, don't we also have issues when passing floats through `is_numeric($value) && false === strpos($value, "\f") && ...` ?

Reply:
we could have a case dedicated to is_float I agree

Reply:
@stof I'm pretty sure that the next case takes care of that already.